static/src/javascripts/bootstraps/standalone.commercial.ts

```diff
@@ -205,15 +209,33 @@ const bootCommercial = async (): Promise<void> => {
 	}
 };
 
+/**
+ * Choose whether to launch Googletag or Opt Out tag (ootag) based on consent state
+ */
+const chooseAdvertisingTag = async () => {
```

Comment:
This is quite a big change to the loading process of commercial. Are we able to analyse any potential performance impact commercial metrics (particularly with Tableau not being available)?

Reply:
As discussed, we'll look to address this in subsequent work, but this isn't a blocker whilst we're still running at zero percent.